### PR TITLE
Lock gems

### PR DESCRIPTION
**Pull request by Mandaryn**
update specs, apply rubocop, remove ruby 1.8, 1.9 support

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# on 2015-10-28 12:04:18 +0100 using RuboCop version 0.34.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 5
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 85
```
Comment (ZenCocoon) [inline .rubocop_todo.yml]: 95 + 5 offense?

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,8 @@
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
   - 2.0.0
   - 2.1.0
   - 2.1.1
```
Comment (ZenCocoon) [inline .travis.yml]: We need to add 2.2.x and 2.3.x though

Comment (Mandaryn) [inline .travis.yml]: good call

File: omniauth-bookingsync.gemspec [hunk 3]
```diff
@@ -8,17 +8,20 @@ Gem::Specification.new do |gem|
...
-  gem.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  gem.executables = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   gem.files = `git ls-files`.split("\n")
   gem.test_files = `git ls-files -- {test,spec,features}/*`.split("\n")
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'omniauth', '~> 1.1'
-  gem.add_dependency 'omniauth-oauth2', '~> 1.1'
+  gem.add_dependency "omniauth", "~> 1.1"
+  # this constraint is to change in 1.4 braking redirect url generation
```
Comment (ZenCocoon) [inline omniauth-bookingsync.gemspec]: typo on `breaking`

File: .travis.yml (hunk 2 above)
Comment (Mandaryn) [inline .travis.yml]: there is no 2.3.x released, yet, nice troll

Comment (ZenCocoon) [inline .travis.yml]: haha meant 2.2.3, my bad ;P

File: README.md [hunk 4]
```diff
@@ -34,7 +36,7 @@ this will skip the account selection process and show the accept/deny page.
 
 ## Supported Rubies
 
-OmniAuth BookingSync is tested under 1.8.7, 1.9.2, 1.9.3, 2.0.0, 2.1.0, 2.1.1, Ruby-head, Ruby Enterprise Edition, JRuby (1.8 and 1.9 mode).
+OmniAuth BookingSync is tested under 2.0.0, 2.1.0, 2.1.1, Ruby-head, Ruby Enterprise Edition, JRuby.
```
Comment (Fredar) [inline README.md]: missing 2.2.3

File: .travis.yml (hunk 2 above)
Comment (Fredar) [inline .travis.yml]: We also seem to use `2.2.0` in HL, booking.com and `2.2.2` at owners app

Comment (Fredar) [inline .travis.yml]: Also nto sure why check for 2.1.0 and 2.1.1 something breaking changed there?
Most of our apps are at 2.1.5 or 2.1.6

Comment (Mandaryn) [inline .travis.yml]: so the rubies should get updated there :P

**Pull request merged**